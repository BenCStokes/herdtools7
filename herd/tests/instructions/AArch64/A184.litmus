AArch64 A184
(* LDXR/STXR Loop, test partial execution, with warnong on stderr *)
{
int x=1;
0:X0=x;
}
  P0             ;
L0:              ;   
 LDXR W1,[X0]    ;
 MOV W3,#2       ;
 STXR W4,W3,[X0] ;
 CBNZ W4,L0      ;
forall (x=2)
