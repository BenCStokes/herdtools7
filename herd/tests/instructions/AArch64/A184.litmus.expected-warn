Warning: File "./herd/tests/instructions/AArch64/A184.litmus", unrolling limit exceeded, legal outcomes may be missing.
