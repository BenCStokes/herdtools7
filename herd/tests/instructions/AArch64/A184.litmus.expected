Test A184 Required
States 1
[x]=2;
Loop Ok
Witnesses
Positive: 3 Negative: 0
Condition forall ([x]=2)
Observation A184 Always 3 0
Hash=79c44439d71d498715a5ee40615de31c

